cli/test_cmd.go

```diff
@@ -114,6 +114,15 @@ func (c *TestCmd) Execute(args []string) error {
 			}
 			expectedDir := filepath.Join(tree, "../../expected", exeMethod, filepath.Base(tree))
 			actualDir := filepath.Join(tree, "../../actual", exeMethod, filepath.Base(tree))
+
+			if _, err := os.Stat(expectedDir); err != nil && os.IsNotExist(err) {
```

Comment:
Actually, can we just remove this clause, and generate expected output for both the program and Docker cases in srclib-java?